environment.yaml

```diff
@@ -1,17 +1,15 @@
 name: openduck_latest
 channels:
-  - bioconda
-  - omnia
   - conda-forge
   - defaults
 dependencies:
   - python=3.9
   - yank=0.25.2
-  - pdbfixer=1.6
-  - openforcefield=0.7.2
+  - pdbfixer=1.7
+  - openff-toolkit=0.9.0
   - parmed=3.4.3
   - cudatoolkit=11.7.0
   - openmm=7.5
-  - openmmforcefields=0.7.4
+  - openmmforcefields=0.9.0
```

Comment:
My attempt to improve the resolution. Basically I removed the defunct omnia channel, and updated all the omnia packages to the oldest possible version available in conda-forge (openforcefield has been renamed to openff-toolkit).

Reply:
I've done some tests on this matter, removing omnia and bioconda quickents the instalation significantly (~5 min resolving the dependencies), albeit some things need to be updated, like all the openforcefield namespaces (to openff-toolkit).

pdbfixer is updated to a newer version too, as the old version is not in conda-forge, but i see no noticeable changes.